src/OmniSharp.MSBuild/ProjectIdInfo.cs

```diff
@@ -13,5 +14,6 @@ public ProjectIdInfo(ProjectId id, bool isDefinedInSolution)
 
         public ProjectId Id { get; set; }
         public bool IsDefinedInSolution { get; set; }
+        public Dictionary<string, string> ConfigurationsInSolution { get; set; }
```

Comment:
Would `SolutionConfiguration` be a better name here?
Consider using `IDictionary` or `IReadOnlyDictionary` instead.

Reply:
I tried to state that this is not 'configurations of solution' but 'configurations of project in solution'. But you are right, `SolutionConfiguration` is indeed a better name.